src/modules/layout/components/Header.js

```diff
@@ -39,7 +38,7 @@ class Header extends React.Component {
               <img src={logoWide} />
             </Link>
           </div>
-          <HeaderNavigationBar className={style.navigationBar} items={this.props.navigationItems} />
+          <HeaderNavigationBar className={style.navigationBar}>{this.props.navigationItems}</HeaderNavigationBar>
```

Comment:
Is it necessary to pass a custom className?

Reply:
Yes, because these are styles that are applied to position the navigationBar and the actionBar inside the header and therefore belongs to the header and not to the bars themselves